### PR TITLE
[idea][bug 473723] Use '\n' as a line separator for the rewritable

### PR DESCRIPTION
**Pull request by akosyakov**
import section

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=473723

**Comment by akosyakov** [comment]
Jenkins intellij-tests-master-latest

File: intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/xbase/idea/imports/IdeaRewritableImportSectionFactory.xtend [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.eclipse.xtext.xbase.imports.RewritableImportSection.Factory
+import org.eclipse.xtext.resource.XtextResource
+
+/**
+ * @author kosyakov - Initial contribution and API
+ */
+class IdeaRewritableImportSectionFactory extends Factory {
+
+	override protected getLineSeparator(XtextResource resource) {
+		'\n'
```
Comment (szarnekow) [inline intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/xbase/idea/imports/IdeaRewritableImportSectionFactory.xtend]: Please add a comment why you did it like that

File: intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/xbase/idea/completion/XbaseCompletionContributor.xtend [hunk 2]
```diff
@@ -11,6 +11,7 @@ import com.google.inject.Inject
 import com.intellij.codeInsight.completion.AllClassesGetter
 import com.intellij.codeInsight.completion.CompletionParameters
 import com.intellij.codeInsight.completion.CompletionResultSet
+import com.intellij.codeInsight.completion.CompletionSorter
```
Comment (szarnekow) [inline intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/xbase/idea/completion/XbaseCompletionContributor.xtend]: Unused import?

File: intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/xbase/idea/imports/IdeaRewritableImportSectionFactory.xtend [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ *******************************************************************************/
+package org.eclipse.xtext.xbase.idea.imports
+
+import org.eclipse.xtext.xbase.imports.RewritableImportSection.Factory
+import org.eclipse.xtext.resource.XtextResource
+
+/**
+ * @author kosyakov - Initial contribution and API
+ */
+class IdeaRewritableImportSectionFactory extends Factory {
```
Comment (szarnekow) [inline intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/xbase/idea/imports/IdeaRewritableImportSectionFactory.xtend]: A better class name would describe its effect, e.g. UnixNewlineIMportSectionFactory or something similar.

**Comment by akosyakov** [comment]
Jenkins intellij-tests-master-latest

**Pull request merged**